test/lit/basic/exception-handling.wast

```diff
@@ -40,7 +100,8 @@
   ;; CHECK-BIN-NEXT:   (local.get $exn)
   ;; CHECK-BIN-NEXT:  )
   ;; CHECK-BIN-NEXT: )
-  (func $exnref-nullexnref-test (result exnref) (local $exn exnref) (local $null-exn nullexnref)
+  (func $exnref-nullexnref-test (result exnref)
+        (local $exn exnref) (local $null-exn nullexnref)
```

Comment:
indentation looks odd?

Reply:
Tried to put that in the separate line because the previous line got too longer... but maybe this looks weirder? Put that back to the previous line. (which now exceeds 80 col, but we don't have policies on column length on tests anyway..)